flaskr/app.py

```diff
@@ -29,7 +29,31 @@ def root():
 
     return app
 
-
+# Instantiate Flask app and SQLAlchemy ORM
 app = create_app()
 db = SQLAlchemy(app)
-from .model import Language
+
+# Define models in relation to instantiated ORM
+class Language(db.Model):
+    __tablename__ = 'languages'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String())
+    methods = db.relationship('Method', backref='language')
+
+    def __repr__(self):
+        return '<Language %r>' % self.name
+
+class Method(db.Model):
+    __tablename__ = 'methods'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    snippet = db.Column(db.Text)
+    syntax = db.Column(db.Text)
+    description = db.Column(db.Text)
+    docs_url = db.Column(db.String)
+    language_id = db.Column(db.Integer, db.ForeignKey('languages.id'))
```

Comment:
Did we decide against importing the way Ian suggested where we wouldn't have to use the `db.` in front of the `Column`?

Reply:
That ended up breaking the migration stuff because the models need to be associated with the particular database associated with the app in the main file. I think there's a way to extract these method definitions out into a separate file, but for now the simplest thing was to do it here so they could be associated with the instance of SQLAlchemy associated with our main app file.

Reply:
That ended up breaking the migration stuff because the models need to be associated with the particular database associated with the app in the main file.

Reply:
It's a particular requirement of the Migration package as far as I can tell, at least if we want the app to auto-create the migration file instead of manually writing it.